app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,7 @@
   <% if @content_item.description %>
     <meta name="description" content="<%= strip_tags(@content_item.description) %>" />
   <% end %>
+  <%= content_pages_nav_test_variant.analytics_meta_tag.html_safe if page_in_scope? %>
```

Comment:
And here?

Reply:
We need the analytics tag if the page is in scope even if we're not loading the navigation.  It lets Haur compare A vs B more easily.